Dockerfile

```diff
@@ -56,11 +56,11 @@ ENV \
   CELLS_WORKING_DIR="/config" \
   GOPATH="/tmp"
 
+COPY --from=buildstage /app/cells /app/
+COPY root/ /
+
 RUN \
+  export PATH=/app/:$PATH && \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
     openssl
-
-COPY --from=buildstage /app/cells /app/
-
-COPY root/ /
```

Comment:
Do not reorganize the structure of the dockerfile (the directory does need to exist when getting added to path).